grakn-graph/src/main/java/ai/grakn/graph/internal/GraknComputerImpl.java

```diff
@@ -91,16 +101,16 @@ private RuntimeException asRuntimeException(Throwable throwable) {
      * @return A graph compute supported by this grakn graph
      */
     @SuppressWarnings("unchecked")
-    protected Class<? extends GraphComputer> getGraphComputer(String graphComputerType) {
+    protected Class<? extends GraphComputer> getGraphComputerClass(String graphComputerType) {
         try {
             return (Class<? extends GraphComputer>) Class.forName(graphComputerType);
```

Comment:
I think Factory gets this information from conf file, so grakn computer may have a spark computer inside or, e.g. a fulgora computer.